src/predicate.rs

```diff
@@ -40,7 +42,21 @@ impl Predicate for Any {
     }
 }
 
-/// Matches Element Node with name `T`.
+/// Matches no Node.
+///
+/// There is no equivalent CSS selector.
+#[derive(Copy, Clone, Debug, PartialEq)]
+pub struct Nothing;
```

Comment:
Is there a real world use of this predicate?

Reply:
Off the top of my head, a function that does some work and then updates the elements given by a selector; if you pass in `Nothing`, then that part of the function won't run.

It's not something in particular I was attached to; I just added it for feature parity.